azurerm/internal/services/securitycenter/server_vulnerability_assessment_resource_test.go

```diff
@@ -0,0 +1,119 @@
+package securitycenter_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type ServerVulnerabilityAssessmentResource struct {
```

Comment:
I assume it's necessary to add a test case for requiresImport, right?

Reply:
On line 32 I have data.ImportStep() that will verify that import works as part of the TestAccServerVulnerabilityAssessment_basic test case.

Reply:
I assume it's not enough. Suggest to add "requiresImport" tc like other [resources](https://github.com/terraform-providers/terraform-provider-azurerm/blob/59066bd464d5f5cce8eddc3e4d5e4f2fb0842bb4/azurerm/internal/services/azurestackhci/stack_hci_cluster_resource_test.go#L34).

Reply:
I will tak a look at this